src/table/components/__tests__/TableHeadWrapper.spec.tsx

```diff
@@ -82,7 +82,17 @@ describe('<TableHeadWrapper />', () => {
     expect(queryByText(tableData.columns[1].label)).toBeVisible();
   });
 
-  it('should call changeSortOrder when clicking a header cell', () => {
+  it.skip('should show the menu button when the head cell is on focus', () => {
+    areBasicFeaturesEnabled = true;
```

Comment:
I'm going to skip this for now. I couldn't verify that the button is visible after focusing the head cell element. The focus is a css focus so I don't see any other option to focus on it than what it is now. It used to work before rebasing the branch. By the way, the current scenario works properly in reality on the table.

Reply:
ok that is fine